examples/hello-world/README.md

```diff
@@ -603,55 +598,63 @@ A strategy's predefined options:
         method: 'post'
       }
     },
-    registerCallback: ({ resolve, body }, username, password, done) => {
+    registerCallback: async ({ resolve, body }, username, password) => {
       // your code to register a new user
-      // you need to call the 'done' callback to notify a user about an error or successful operation
     },
-    loginCallback: ({ resolve, body }, username, password, done) => {
+    loginCallback: async ({ resolve, body }, username, password) => {
       // your code to implement a user login
-      // you need to call the 'done' callback to notify a user about an error or successful operation
     },
     failureCallback // default behavior
   })
   ```
-
+    
 * #### githubStrategy
   ```js
-  githubStrategy({
+  resolveAuth(PassportGithubStrategy, {
     strategy: {
       clientID: 'MyClientID',
       clientSecret: 'MyClientSecret',
       callbackURL: 'http://localhost:3000/auth/github/callback',
       successRedirect: null
     },
     routes: {
-      auth: '/auth/github',
-      callback: '/auth/github/callback'
+      auth: {
+        path: '/auth/github',
+        method: 'get'
+      },
+      callback: {
+        path: '/auth/github/callback',
+        method: 'get'
+      }
     },
-    authCallback: ({ resolve, body }, profile, done) => {
+    authCallback: async ({ resolve, body }, profile) => {
       // your code to authenticate a user
-      // you need to call the 'done' callback to notify a user about an error or successful operation
     },
     failureCallback // default behavior
```

Comment:
Please, change to worked empty pattern, I guess it should be

failureCallbask: (e) => {
    console.log(e)
}